test/Riok.Mapperly.Tests/_snapshots/MapperTest.SameMapperNameInMultipleNamespacesShouldWork.01FooBarMapper1.g.verified.cs

```diff
@@ -0,0 +1,12 @@
+﻿//HintName: FooBarMapper1.g.cs
+#nullable enable
+namespace Test.B
+{
+    internal partial class FooBarMapper
```

Comment:
The leading `01` in the file name is caused by the test framework, right?

Reply:
It is.